src/pages/admin/manageclubs.js

```diff
@@ -0,0 +1,110 @@
+import { gql, useQuery, useMutation } from "@apollo/client";
+import { Box, Grid, Snackbar } from "@mui/material";
+import { useState } from "react";
+import OrganizationPicker from "../../comps/ui/OrganizationPicker";
+import AdminMeetingCard from "../../comps/pages/admin/AdminMeetingCard";
+
+const classes = {
+	mainDiv: {
+		width: "1200px",
+		maxWidth: "90%",
+		margin: "auto"
+	}
+};
+
+const MUTATION = gql`
+	mutation ($meetingId: Int!) {
+		adminDeleteMeeting(meetingId: $meetingId)
+	}
+`;
+
+const QUERY = gql`
+	query ($id: Int!) {
+		organization: organizationById(id: $id) {
+			id
+			name
+			url
+			memberships {
+				id
+				user {
+					id
+					name
+					firstName
+					lastName
+					email
+					isFaculty
+					fourDigitId
+				}
+				role
+				adminPrivileges
+			}
+			updates {
+				title
+				content
+			}
+			meetings {
+				id
+				title
+				description
+				start
+				end
+				privacy
+				rooms {
+					id
+					name
+					floor
+					approvalRequired
+				}
+			}
+		}
+	}
+`;
+
+const ManageClubs = () => {
+	const [orgId, setOrgId] = useState(0);
+	const { data } = useQuery(QUERY, {
+		variables: {
+			id: orgId
+		}
+	});
+
+	let [message, setMessage] = useState("");
+
+	const [deleteMeeting] = useMutation(MUTATION);
+
+	return (
+		<Box sx={classes.mainDiv}>
+			<OrganizationPicker setOrgId={setOrgId} />
+			<Grid container>
+				{data &&
+					data.organization &&
+					data.organization.meetings.map((meeting, i) => (
+						<Grid item xs={12} sm={12} md={6} lg={6} xl={4}>
+							<AdminMeetingCard
```

Comment:
Can't seem to get this to show up on https://beta.stuyactivities.org/admin/manageclubs, even after adding a new meeting with a room set in the future.